ament_tools/verbs/list_dependencies.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from ament_tools.helper import argparse_existing_dir
+from ament_tools.packages import find_packages
+
+
+def prepare_arguments(parser):
+    parser.add_argument(
+        '--basepath',
+        type=argparse_existing_dir,
+        default=os.curdir,
+        help='Base paths to recursively crawl for packages',
+    )
+    parser.add_argument(
+        '--build-deps',
+        action='store_true',
+        help='Show only build dependencies of a given package'
+    )
+    parser.add_argument(
+        '--doc-deps',
+        action='store_true',
+        help='Show only dependencies to generate documentation of a given package'
+    )
+    parser.add_argument(
+        '--exec-deps',
+        action='store_true',
+        help='Show only exec dependencies of a given package'
+    )
+    parser.add_argument(
+        '--test-deps',
+        action='store_true',
+        help='Show only test dependencies of a given package'
+    )
+    parser.add_argument(
+        'package',
+        metavar='PACKAGE',
+        help='Package to process'
+    )
+    return parser
+
+
+def main(options):
+    deps = []
+    packages = find_packages(options.basepath)
+    # show all dependencies if no options are given
+    if not (options.build_deps or options.doc_depends or options.exec_deps or options.test_deps):
```

Comment:
This should be `doc_deps` to match the argument defined in the parser, same thing line 63